appveyor.yml

```diff
@@ -1,43 +1,65 @@
 # Configuration for continuous integration service at appveyor.com
 
-shallow_clone: true
+version: '{build}'
 
-# Operating system (build VM template)
 os: Visual Studio 2015
 
-# scripts that are called at very beginning, before repo cloning
-init:
-
-# clone directory
-clone_folder: c:\projects\gflags
-
-matrix:
-  fast_finish: true
+environment:
+  matrix:
+  - Toolset: v140
+  - Toolset: v120
+  - Toolset: v110
+  - Toolset: v100
```

Comment:
On it, was so far just figuring out how to have multiple toolchains with AppVeyor...